srcpkgs/nvimpager/template

```diff
@@ -10,15 +10,14 @@ build_style=gnu-makefile
 make_build_target="nvimpager.configured"
 make_build_args="PREFIX=/usr"
 hostmakedepends="neovim $(vopt_if man pandoc)"
-depends="neovim>=0.3.0 bash"
+depends="neovim>=0.4.0 bash"
 short_desc="Wrapper to use neovim as a pager to view manpages, diffs, etc"
 maintainer="Adrian Herath <adrianisuru@gmail.com>"
 license="BSD-2-Clause"
 homepage="https://github.com/lucc/nvimpager"
 distfiles="${homepage}/archive/v${version}.tar.gz
  https://www.vim.org/scripts/download_script.php?src_id=14498>AnsiEsc.vba.gz"
```

Comment:
Should we update this at some point?

Reply:
Which "this" are you talking about? The depends? Sorry I'm not clear on what you are asking.

Reply:
Sorry, I meant `src_id=14498` in the link.

Reply:
So, given https://github.com/lucc/nvimpager/blob/develop/makefile , I don't see this file being used anymore? Maybe we should remove from package.

Reply:
We should definitely remove, they no longer use it.